### PR TITLE
Add uptime stats

### PR DESCRIPTION
**Pull request by iiiiii1wepfj**

### CODE REVIEW

File: eduu/__main__.py [hunk 1]
```diff
@@ -34,6 +35,7 @@ async def main() -> None:
 
     client.me = await client.get_me()
 
+    client.eduurobot_bot_start_time = time.time()
```
Comment (AndrielFR) [inline eduu/__main__.py:38]: Could you change from `client.eduurobot_bot_start_time` to `client.start_time`?

**Review by AndrielFR — APPROVED** [review]

File: eduu/__main__.py (hunk 1 above)
Comment (iiiiii1wepfj) [reply eduu/__main__.py:38]: @AndrielFR but i don&#x27;t know if they use this name in the library

Comment (AndrielFR) [reply eduu/__main__.py:38]: no they don't use

Comment (iiiiii1wepfj) [reply eduu/__main__.py]: @AndrielFR ok, done

Comment (AndrielFR) [reply eduu/__main__.py]: Better. 😁

**Comment by alissonlauffer** [comment]
What about using datetime's timedelta for this? It might not show the time in the most optimal way, but at least no external dependency is required (as I avoid using external dependencies for "simple" things like this):

```python
>>> from datetime import datetime
>>> 
>>> start_time = datetime(2021, 5, 26, 17, 0)
>>> 
>>> print(datetime.now() - start_time)
59 days, 22:37:35.606562
>>>
```

**Comment by alissonlauffer** [comment]
Right, so the `humanfriendly` library does not pull any other external dependency, so it looks ok to be added. Just let me do some changes before merging.

**Review by alissonlauffer — APPROVED** [review]

**Pull request merged**